pep-0008.txt

```diff
@@ -850,6 +850,14 @@ names.
 In some fonts, these characters are indistinguishable from the
 numerals one and zero.  When tempted to use 'l', use 'L' instead.
 
+While support for Unicode identifiers has been added to allow all
+Python developers to use variable names in their native languages,
+the 7-bit ASCII text encoding remains the lowest common denominator
+for text entry when developing Python programs. Accordingly, all
+public standard library APIs use ASCII-only identifiers, and the
+reference implementation's use of non-ASCII identifiers is limited
+to appropriately testing that functionality.
+
```

Comment:
+1 to the general text, but what about putting the admonition at the front of the paragraph?  E.g.

```
Use ASCII-only identifiers in stdlib public APIs.  While support for...
```